seed/models/analyses.py

```diff
@@ -1,21 +1,46 @@
-
 from django.contrib.postgres.fields import JSONField
 from django.db import models
 
 from seed.landing.models import SEEDUser as User
+from seed.lib.superperms.orgs.models import Organization
 
 
 class Analysis(models.Model):
-
+    """
+    The Analysis represents an analysis performed on one or more properties.
+    """
     BSYNCR = 1
 
-    ANALYSIS_TYPES = (
+    SERVICE_TYPES = (
         (BSYNCR, 'BSyncr'),
     )
 
-    name = models.CharField(max_length=255, blank=False, default=None)
```

Comment:
Oops - We should have a `name` on this model as well. That was our bad missing it in the initial diagrams 😬

Reply:
good catch, I'll add that real quick